pkg/istio-agent/sds-agent.go

```diff
@@ -128,7 +131,7 @@ const (
 
 var (
 	// LocalSDS is the location of the in-process SDS server - must be in a writeable dir.
-	LocalSDS = "/etc/istio/proxy/SDS"
+	LocalSDS = "./etc/istio/proxy/SDS"
```

Comment:
does the corresponding field in bootstrap not need to change as well? I did the same in https://github.com/istio/istio/pull/22679

Reply:
Why do we need "."? What would happen without "."?